vnfs/libexec/wwmkchroot/functions

```diff
@@ -245,13 +273,20 @@ configure_services() {
 
 cleanup() {
     if [ -n "$INSTALL_ISO" ]; then
-        for i in `ls -d $MEDIA_MOUNTPATH.*`; do
-            if umount $i; then
-                rmdir $i
-            else
-                echo "ERROR: Could not unmount $i"
-            fi
-        done
+        if [ -n "$DEBUG" ]; then
+            echo "WARNING: Debug enabled; leaving media mounts in place!"
+        else
+            rm -f $MEDIA_TMP/*
```

Comment:
You're using `$MEDIA_TMP/*` here ... Above you're setting:
```
MEDIA_MOUNTPATH="${CHROOTDIR}${MEDIA_TMP}"
```
Which is to be used?

Reply:
It's actually both. I added some comments to the variable declaration that will hopefully clarify this. Basically, everything is mounted under the chroot, but the host need to access the media to bootstrap the image. The same directory is created on the host with links to the media in the chroot.

In case you're wonding why I make the symlinks to the files: I could have just created one symlink to the media directory, but I would need remove any directory with that name first which seemed too destructive for me.